ams/server/src/main/java/com/netease/arctic/server/persistence/SqlSessionFactoryProvider.java

```diff
@@ -127,6 +132,9 @@ private void createTablesIfNeed(Configurations config) {
         query = String.format(
             "SELECT 1 FROM information_schema.tables WHERE table_schema = '%s' AND table_name = '%s'",
             connection.getCatalog(), "CATALOG_METADATA");
+      } else if (ArcticManagementConf.DB_TYPE_POSTGRES.equals(dbTypeConfig)) {
+        query = "SELECT 1 FROM information_schema.tables WHERE table_schema = 'public' AND table_name = " +
+            "'catalog_metadata'";
```

Comment:
Is the database information missing? And is the schema fixed as public?

Reply:
I think database informantion has been set in jdbc url

Reply:
So remove condition: `table_schema = 'public'` is better solution?

Reply:
Is there someway to get the current table_schema ?

Reply:
Yes, execute SQL `SELECT current_schema()` will retuen the value

Reply:
So we can change the sql to `WHERE table_schema = current_schema() AND table_name = 'catalog_metadata' `

Reply:
Fixed